src/styles/search.css

```diff
@@ -0,0 +1,26 @@
+.search-form {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  position: relative;
```

Comment:
I dont think you need to `flex-direction` and `position` properties adding in here, you can remove these without any ill effects to behaviour (remember flex-direction is set to row by default, so you dont need to explicitly state like this unless you need to override a previous declaration)